core/domain/classifier_domain.py

```diff
@@ -125,46 +125,123 @@ def __init__(self, job_id, algorithm_id, interaction_id, exp_id,
 
     @property
     def job_id(self):
+        """Returns the job_id of the classifier training job.
+
+        Returns:
+            str. The unique id of the classifier training job.
+        """
         return self._job_id
 
     @property
     def algorithm_id(self):
+        """Returns the algorithm_id of the algorithm used for generating
+            the classifier.
```

Comment:
Deindent by 4. Ditto elsewhere. (We only indent the continuation lines for Args and Raises.)